server/command/service.go

```diff
@@ -30,52 +30,138 @@ type service struct {
 
 var _ Service = (*service)(nil)
 
-func MakeService(mm *pluginapi.Client, configService config.Service, proxy proxy.Service, httpOut httpout.Service) (Service, error) {
-	s := &service{
-		mm:      mm,
-		conf:    configService,
-		proxy:   proxy,
-		httpOut: httpOut,
-	}
-	conf := configService.GetConfig()
-	subCommands := s.getSubCommands()
-	var subTrigger []string
-	for t, c := range subCommands {
-		if c.debug && !conf.DeveloperMode {
-			continue
-		}
+type params struct {
+	pluginContext *plugin.Context
+	commandArgs   *model.CommandArgs
+	current       []string
+}
+
+type commandHandler struct {
+	f            func(*params) (*model.CommandResponse, error)
+	devOnly      bool
+	autoComplete *model.AutocompleteData
+}
+
+func (s *service) allCommands() map[string]commandHandler {
+	uninstallAC := model.NewAutocompleteData("uninstall", "", "Uninstall an app")
+	uninstallAC.AddTextArgument("ID of the app to uninstall", "appID", "")
 
-		subTrigger = append(subTrigger, t)
+	all := map[string]commandHandler{
+		"info": {
+			f:            s.executeInfo,
+			autoComplete: model.NewAutocompleteData("info", "", "Display debugging information"),
+		},
+		"list": {
+			f:            s.executeList,
```

Comment:
Given that is coding is getting touched: Should `s.checkSystemAdmin` be added here for conistency?